src/unk_0203E880.c

```diff
@@ -739,84 +603,74 @@ static u16 Unk_02100AD4[] = {
     0xDE
 };
 
-void sub_0203F310 (FieldSystem * fieldSystem)
+void FieldSystem_ClearDailyHiddenItemFlags (FieldSystem * fieldSystem)
```

Comment:
**question:** How is this invoked? It seems like it operates on maps that... *aren't* the Iron Island / Floaroma Meadow maps?

Reply:
It's called in the function that runs once a day to update day-based events. My guess is it's written that way so hidden items don't reappear on any maps that you're currently located in.